config/initializers/1_fast_gettext.rb

```diff
@@ -15,6 +15,11 @@
 # work in all domains context by default (for plugins)
 include FastGettext::TranslationMultidomain
 
+# Keep TRANSLATORS comments
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=TRANSLATORS:]
+# Disable fuzzy .po merging
+Rails.application.config.gettext_i18n_rails.msgmerge = %w[--no-fuzzy-matching]
```

Comment:
Are you referring to this?

<pre>
 +#, fuzzy
+msgid ""
+msgstr ""
</pre>

If so, I'd suggest just removing the empty string from where you're extracting it (probably https://github.com/theforeman/foreman_remote_execution/pull/145/files#diff-4354b6d435b32105d6decf507c6bca49R7).  It's unclear that you're actually exercising the code mentioned here if you're not doing a pull and merge.

As an aside, I'll try and review that PR as the strings are not good enough to translate.